spec/controllers/projects/contributions_controller_spec.rb

```diff
@@ -110,6 +110,25 @@
     end
   end
 
+  describe 'unavailable contribution projects' do
+    let(:project) { create(:project, available_for_contribution: false) }
+    let(:reward)  { create(:reward, project: project) }
+
+    context 'new' do
```

Comment:
This case is related with a type of Project which is not available for contributions made via the juntos' platform(maybe this not make sense, but it was a costumer's request). There is another case for blocking contributions too, when the project have a state different of 'online'. This is the first hook(```project_is_online?```)

I guess that the other test case which you suggested me to do is already done in the ```post``` context. Or am I wrong?